### PR TITLE
Applied fixes from StyleCI

### LINKED ISSUE #223: Update StyleCI
There is a config in StyleCI which allows you to set file headers. 

We should use that to force the following header: 

```php
/*
 * Copyright (C) 2013-2016 Mailgun
 *
 * This software may be modified and distributed under the terms
 * of the MIT license.  See the LICENSE file for details.
 */
```

### PR DESCRIPTION
**Pull request by pirogoeth**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/XWj1oN).

---

Adds header comments to all PHP files. Fixes #223.

### CODE REVIEW

**Comment by pirogoeth** [comment]
paging @Nyholm

**Comment by Nyholm** [comment]
Awesome. Great!

File: src/Mailgun/Api/HttpApi.php [hunk 1]
```diff
@@ -1,5 +1,12 @@
 <?php
 
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
```
Comment (Nyholm) [inline src/Mailgun/Api/HttpApi.php]: Oh no, There are double space here

Comment (Nyholm) [inline src/Mailgun/Api/HttpApi.php]: For reference. Symfony is using: 

```php

/*
 * This file is part of the Symfony package.
 *
 * (c) Fabien Potencier <fabien@symfony.com>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
```

**Pull request merged**